### ISSUE TITLE
[A11y]For the links present in 'Manage organisation' page color is used as only means to convey information.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8488]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1294150

## ​Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Select “Manage Organizations” from the Profile Dropdown and hit enter.
Tab till links present below the  and observe screen color is used as only means to convey information.

## Actual:​
For the links present in 'Add-on' page color is used as only means to convey information and color contrast ratio is 2.1 that is below 3:1.

This issue is observed throughout the application for the links.

## Expected:​
Color should not be used as only means to convey information about links. 'Underline' should provide below link and and color contrast ratio should be equal or above 3:1.

​## User Impact:​
Color blindness users may find difficulty in distinguishing links from normal text.

### Notes:
I was unable to reproduce this. As far as I can tell, the link is the correctly colour. Please verify that the contrast is correct on this (it is possible that the colour may not meet contrast requirements for visited links?)
